internal/testkeys/testkeys.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2025 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package testkeys
+
+import (
+	"crypto"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+)
+
+type EncodedKeyString string
```

Comment:
What's the idea behind this type def?

Reply:
Nevermind, I get now that this says "I'm really sure that this is a PEM-encoded key and not a random other string".